### PR TITLE
Feature/test-create

### PR DESCRIPTION
**Pull request by devalexandre**
test implementation proposal.

```
npx jest test/**/*.spec.js --detectOpenHandles --forceExit
```

### CODE REVIEW

File: test/e2e/create/create.spec.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			.usage("Usage: $0 <command> [options]")
+			.version()
+			.command(create).default('--typescript',true)
+			.help()
+			.argv;
+
+		const parser = new YargsPromise(yargs);
+		await parser.parse(`create service ${answers_ts.serviceName} --typescript `);
+		expect(fs.existsSync(_path)).toBeTruthy();
+		fs.unlinkSync(_path);
```
Comment (icebob) [inline test/e2e/create/create.spec.js:52]: Please check the content of the file, not just the existing.

**Review by icebob — CHANGES_REQUESTED** [review]

File: test/e2e/create/create.spec.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		yargs
+			.usage("Usage: $0 <command> [options]")
+			.version()
+			.command(create)
+			.help()
+			.argv;
+		const parser = new YargsPromise(yargs);
+		await parser.parse(`create ${answers.serviceName}`);
+		expect(fs.existsSync(_path)).toBeTruthy();
+		fs.unlinkSync(_path);
```
Comment (icebob) [inline test/e2e/create/create.spec.js:28]: Same here

**Review by devalexandre — COMMENTED** [review]
checked files content

File: test/e2e/create/create.spec.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+			.version()
+			.command(create).default('--typescript',true)
+			.help()
+			.argv;
+
+		const parser = new YargsPromise(yargs);
+		await parser.parse(`create service ${answers_ts.serviceName} --typescript `);
+		expect(fs.readFileSync(_path)).toEqual(fs.readFileSync(_mock_file))
+
+		expect(fs.existsSync(_path)).toBeTruthy();
```
Comment (icebob) [inline test/e2e/create/create.spec.js:58]: Would be better to change the order of `expect`. IF the content is same, not relevant to check the existing. But if the file is not exist, no need to check the content.

**Review by icebob — CHANGES_REQUESTED** [review]

**Comment by icebob** [comment]
The `npm test` doesn't work.

image

**Review by icebob — APPROVED** [review]
Thanks.

**Pull request merged**